partner_street_number/static/src/css/field.css

```diff
@@ -0,0 +1,11 @@
+.o_address_street_name {
+	   float: left !important;
+       margin-right: 2% !important;
+       width: 75% !important;
+   }
+
+.o_address_street_number {
+       margin-right: 0% !important;
+       width: 25% !important;
```

Comment:
is is possible to avoid the `!important`s by adding a longer selector? The important qualifier tends to mess up css in the sense that it overrides everything